app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -799,7 +799,7 @@ def rbac_field_changed(rec_type)
     when "role"  then rbac_role_get_form_vars
     end
 
-    changed = (@edit[:new] != @edit[:current])
+    session[:changed] = changed = (@edit[:new] != @edit[:current])
```

Comment:
not sure, the handling of the tab change is in different method, so anyway it needs to be loaded then in ```get_session_data``` when user is changing tabs. If you don't want to put in session object then I can do the check ```@edit[:new] != @edit[:current]``` inside the method loading the tabs https://github.com/ManageIQ/manageiq-ui-classic/pull/1196/files#diff-caf44955008588ca9bc62b9633fbdb9cR204